### PR TITLE
Objective-c snippets added

### PR DESCRIPTION
**Pull request by maratal**
Not sure whether weakSelf/strongSelf objective-c practice should be used here. Because it makes snippets very noisy and hard to read. On the other hand it's very bad to teach someone not to use it. What do you think?

### CODE REVIEW

**Comment by lmars** [comment]
I'll defer to others for a review since I'm unfamiliar with Obj-C

**Comment by lukasz-szyszkowski** [comment]
We should promote good practice, so +1 from my site for weak self

**Comment by uxduck** [comment]
> We should promote good practice, so +1 from my site for weak self

I agree, but there was discussion about a different set of snippets (the flutter one) being too long to fit nicely on the webpage. So you may have the same issue here too. @fliptopbox , can you take a look if these snippets are too big?

File: objc/live-updates-pub.m [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ARTRealtimeChannel *channel = [ably.channels get:@"dogecoin:usd"];
+[channel publish:@"rate" data:@{ @"price": @1.01, @"time": @([[NSDate date] timeIntervalSinceReferenceDate]) }];
```
Comment (uxduck) [inline objc/live-updates-pub.m]: Why is there `@` in front of `1.01` and wrapping `[[NSDate date] timeIntervalSinceReferenceDate]`? I guess it's objective-C syntax, but the example [here](https://stackoverflow.com/a/41868708/7365866) doesn't use it.

Comment (lukasz-szyszkowski) [reply objc/live-updates-pub.m]: in objective-c `@` is literal for NSNumber object, it's the same as `[NSNumber numberWithFloat:1.01];`
more [HERE](https://clang.llvm.org/docs/ObjectiveCLiterals.html)

Comment (uxduck) [reply objc/live-updates-pub.m]: Okay great, thanks

**Review by uxduck — APPROVED** [review]

**Review by lukasz-szyszkowski — APPROVED** [review]

**Comment by jamienewcomb** [comment]
Spoke to @maratal -  MQTT example to be updated and weak self to be removed (these are for marketing purposes only)

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**